pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1325,7 +1325,7 @@ cdef class BusinessDay(BusinessMixin):
                 off_str += str(td.microseconds) + "us"
             return off_str
 
-        if isinstance(self.offset, timedelta):
+        if PyDelta_Check(self.offset):
```

Comment:
Cython doesn't optimize this automatically? 
(and does this make a noticeable difference?)

Reply:
cython does not do this automatically.  it makes a small differnece, but enough that we do this in the rest of the cython code

Reply:
OK

Reply:
yeah we use this type of check with datetime types a lot